domain/user/errors/errors.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package errors
+
+import (
+	"github.com/juju/errors"
+)
+
+const (
+	// NotFound describes an error that occurs when the user being request does
+	// not exist.
+	NotFound = errors.ConstError("user not found")
+
+	// UserCreatorNotFound describes an error that occurs when a users creator
```

Comment:
"user's". And you don't need the parentheses bit for clarification.

Reply:
fixed